apps/src/studio/studio.js

```diff
@@ -399,6 +399,13 @@ var drawMap = function () {
   }
   Studio.applyGoalEffect();
 
+  // Create cloud elements.
+  for (i = 0; i < constants.NUM_CLOUDS; i++) {
+    var cloud = document.createElementNS(SVG_NS, 'image');
+    cloud.setAttribute('id', 'cloud' + i);
+    svg.appendChild(cloud);
```

Comment:
I feel like the clouds should be added to a group/layer instead of being children of the SVG directly.  It would help with sorting and/or applying effects in the future.